tests/framework/framework.go

```diff
@@ -96,6 +96,23 @@ func init() {
 	master = flag.String("master", "", "master url:port")
 }
 
+// NewFrameworkOrDie calls NewFramework and handles errors by calling Fail. Config is optional, but
+// if passed there can only be one.
+func NewFrameworkOrDie(prefix string, config ...Config) *Framework {
+	if len(config) > 1 {
+		Fail("only 1 config can be passed to NewFrameworkOrDie")
+	}
+	cfg := Config{}
+	if len(config) == 1 {
```

Comment:
what if config isn't passed in at all?

Reply:
Then len(config) == 0 and cfg == `Config{}` an empty Config struct.